### PR TITLE
How to setup agent hooks 

### PR DESCRIPTION
**Pull request by nsuma8989**
This PR adds table of content for the readme and also added new section for how to setup agent hooks on agent-stack-k8s

### CODE REVIEW

**Review by zhming0 — APPROVED** [review]
🚀

File: README.md [hunk 1]
```diff
@@ -437,6 +457,164 @@ This currently can't prevent every sort of error, you might still have a referen
 
 Our JSON schema can also be used with editors that support JSON Schema by configuring your editor to validate against the schema found [here](./cmd/linter/schema.json).
 
+## How to setup agent hooks
+
+This section explains how to setup agent hooks when running Agent Stack K8s. In order for the agent hooks to work, as we all know we need them to be present on the instances where the agent runs.
```
Comment (DrJosh9000) [inline README.md:462]: "as we all know" - _do_ we all know?

```suggestion
This section explains how to setup agent hooks when running Agent Stack K8s. In order for the agent hooks to work, they must be present on the instances where the agent runs.
```

**Review by DrJosh9000 — APPROVED** [review]
LGTM, I have one question phrased as a suggestion (I'm not a qualified technical writer, so happy to be overridden)

**Comment by nsuma8989** [comment]
@zhming0 @DrJosh9000 Thank you for the review and approvals. Please can you help with the merging of the PR as build is not getting triggered since it is on a branch from my fork

**Pull request merged**